webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -146,7 +155,50 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
     }
   }, [canceler.signal, onTrialUpdate, trialId]);
 
-  const { stopPolling } = usePolling(fetchTrialDetails, { rerunOnNewFn: true });
+  const fetchRemainingLogDays = useCallback(async () => {
+    if (!trialId) return;
+    try {
+      const remainingDays = await getTrialRemainingLogRetentionDays({ id: trialId });
+      setRemainingLogDays(
+        remainingDays.remainingLogRetentionDays !== undefined
+          ? Loaded(remainingDays.remainingLogRetentionDays)
+          : NotLoaded,
+      );
+    } catch (e) {
+      setRemainingLogDays(NotLoaded);
+    }
+  }, [trialId]);
+
+  const fetchAllTrialDetails = useCallback(async () => {
+    await Promise.allSettled([fetchTrialDetails(), fetchRemainingLogDays()]);
```

Comment:
please make sure to check the status with allSettled https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled